display/json/json.go

```diff
@@ -24,7 +24,12 @@ import (
 
 func Display(object interface{}) error {
 	if bytes, e := json.Marshal(object); e == nil {
-		fmt.Printf("%v\n", string(bytes))
+		content := string(bytes)
+		if content == "null" {
+			fmt.Print("[]\n")
```

Comment:
the `object` is not necessarily always `array`

Reply:
agree